internal/dbsqlc/river_job.sql

```diff
@@ -72,6 +72,7 @@ WITH locked_jobs AS (
   WHERE
     state = 'available'::river_job_state
     AND queue = @queue::text
+    AND scheduled_at <= now()
```

Comment:
This should be totally fine from a query performance perspective because it's part of the compound fetching index and is already being used for sorting.

Reply:
Great. Yeah I figured that the extra predicate is on average going to have very little work to do since the vast majority of rows that aren't ready yet will still be `scheduled`.